projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/serverRequests.ts

```diff
@@ -65,7 +65,7 @@ export async function jobRunRequest(): Promise<{
  * Sent a POST request to the server to execute a VDK operation a data job.
  * The information about the data job is retrieved from jobData object and sent as JSON.
  */
-export async function jobRequest(endPoint: string): Promise<void> {
+export async function jobRequest(endPoint: string): Promise<string> {
```

Comment:
Update the method description to specify the new contract of the method 
If there is an error it returns empty string otherwise returns message to print.

Though to be honest a bit more structured and explicit reutnr type would be better 

WHat if we want to print error message ?
`{ status="sucess|failure", message = "" } `

Reply:
@antoniivanov yes, you are right i a changing it to  send a similar object instead of just string